pkg/client/unversioned/request.go

```diff
@@ -244,11 +246,10 @@ func (r *Request) AbsPath(segments ...string) *Request {
 	if r.err != nil {
 		return r
 	}
-	if len(segments) == 1 {
+	r.path = path.Join(r.proxyPrefix, path.Join(segments...))
```

Comment:
well, unit tests are passing when I do that (after refactoring) so that looks promising. going to smoke test a bit and then push my latest changes